### ISSUE TITLE
Feature request: Re-run last test

### ISSUE THREAD

**Issue by qnkhuat** [Olical/conjure#558]
Thank you for the fantastic plugin :)

One of the things that I do often is that I need to re-run a test while making changes.
It'd save me a lot of file switching if we have something similar to `ep` but for test.

I'd expect the command to re-run the last test that was triggered.

I'm happy to open a PR if you can give me some pointers.
Thanks

**Comment by Olical** [comment]
Good idea! I guess it'd be good to check if the testing APIs have a way to do this in a first class way, rather than Conjure having to keep the state somewhere. I'm happy to keep the state of the last test ran, we should just check if there's a built in mechanism to CIDER or something that already does this that we can hook into. I have a feeling there is?

**Comment by Olical** [comment]
Ah I think CIDER supports running failed tests, not the last test you ran, so maybe we would have to implement it.
